src/containers/Home/index.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import Helmet from 'react-helmet';
+import IndexHeader from 'components/header/IndexHeader';
+import {NavLink} from 'fluxible-router';
+import {connectToStores, provideContext} from 'fluxible-addons-react';
+import SurahsStore from 'stores/SurahsStore';
+import UserStore from 'stores/UserStore';
+import debug from 'utils/Debug';
+
+const styles = require('./style.scss');
```

Comment:
This is pure :sparkles: 

What is happening here is we are including a CSS file which contains classes. On compile each class gets assigned a random hash and can be used in the JS file. The hash corresponds to that style of the class, and therefore is _scoped_ because classes are not shared across files (so the `.title` class won't be the same hash in `Home/index.js` as `Surah/index.js` for example). This is awesome and where this world is moving to.